benchmarks/run-benchmark.py

```diff
@@ -43,6 +43,14 @@
     "tpcds-1gb-parquet": ParquetTPCDSBenchmarkSpec(scale_in_gb=1),
     "tpcds-3tb-parquet": ParquetTPCDSBenchmarkSpec(scale_in_gb=3000),
 
+    # Merge data load
```

Comment:
nit: Punctuation missing. Same for other comments below.

Reply:
Done